app/models/miq_report/generator/html.rb

```diff
@@ -156,7 +156,7 @@ def build_group_html_rows(group, col_count, label = nil, group_text = nil)
         if extras[:grouping][group].key?(calc.first) # Only add a row if there are calcs of this type for this group value
           grp_output = ""
           grp_output << "<tr>"
-          grp_output << "<td#{in_a_widget ? "" : " class='group'"} style='text-align:right'>#{calc.last.pluralize}:</td>"
+          grp_output << "<td#{in_a_widget ? "" : " class='group'"} style='text-align:right'>#{_(calc.last.pluralize)}:</td>"
```

Comment:
code like this is difficult to test / detect when translations are added or removed.
It is not detectable statically which also means it is not grep friendly.

Reply:
@kbrock this feels problematic.  Where does `calc.last` come from?  /cc @mzazrivec

Reply:
@chessbyte `calc.last` comes from `MiqReport::GROUPINGS`:
```ruby
GROUPINGS = [[:min, N_("Minimum")], [:avg, N_("Average")], [:max, N_("Maximum")], [:total, N_("Total")]].freeze
```

The strings in that constant are properly marked for string collection, so I think we're good here.

Reply:
Actually, no. I spoke too soon. Since we do `_(calc.last.pluralize)`, we're applying gettext to plural forms, but what we collect from `MiqReport` are singular forms (i.e. we don't have the plural forms in the gettext catalog).

So we probably need to modify the `MiqReport` class and that constant to also include plural forms (and adjust the code using it accordingly).